apps/grader-host/src/index.js

```diff
@@ -364,16 +367,14 @@ async function runJob(context, receivedTime, tempDir) {
     job: { jobId, image, entrypoint, timeout, enableNetworking, environment },
   } = context;
 
-  let results = {};
-  let runTimeout = timeout || config.defaultTimeout;
   // Even if instructors specify a really short timeout for the execution of
   // the grading job, there's a certain amount of overhead associated with
   // running the job (pulling an image, uploading results, etc.). We add a
   // fixed amount of time to the instructor-specified timeout to account for
   // this.
-  let jobTimeout = config.timeoutOverhead + runTimeout;
-  let jobEnableNetworking = enableNetworking || false;
-  let jobEnvironment = environment || {};
```

Comment:
These fallbacks existed because at some point we had to add them while maintaining backwards compatibility with messages coming from PrairieLearn that didn't have them. It's been years since then, I think we can safely expect these properties to always be present (and Zod will enforce that they _are_ present long before we get here).